lib/module-utils.js

```diff
@@ -23,13 +23,15 @@ function getModuleInfo(filePath) {
 }
 
 function normalizeScriptPath(scriptPath) {
+  let normalizedScriptPath = scriptPath;
+
   // Remove file prefix which was added in Node v10.12
   if (scriptPath.startsWith('file://') && path.sep === '/') {
-    scriptPath = scriptPath.substring('file://'.length);
+    normalizedScriptPath = scriptPath.substring('file://'.length);
```

Comment:
Why not just `return`? We're not logging the value.

Reply:
can't say I feel too strongly about it, I can change it if you think it's more readble

Reply:
resolving the conversation without making the change. if you think it's more readable the way you suggested I'm more than happy to make the change!